src/main/java/net/neoforged/neoforge/common/data/internal/NeoForgeItemTagsProvider.java

```diff
@@ -130,6 +130,13 @@ public void addTags(HolderLookup.Provider lookupProvider) {
                         Tags.Items.FOODS_RAW_MEAT, Tags.Items.FOODS_RAW_FISH, Tags.Items.FOODS_COOKED_MEAT, Tags.Items.FOODS_COOKED_FISH,
                         Tags.Items.FOODS_SOUP, Tags.Items.FOODS_CANDY, Tags.Items.FOODS_GOLDEN,
                         Tags.Items.FOODS_EDIBLE_WHEN_PLACED, Tags.Items.FOODS_FOOD_POISONING);
+        tag(Tags.Items.ANIMAL_FOODS)
+                .addTags(ItemTags.SNIFFER_FOOD, ItemTags.PIGLIN_FOOD, ItemTags.FOX_FOOD, ItemTags.COW_FOOD,
+                        ItemTags.GOAT_FOOD, ItemTags.SHEEP_FOOD, ItemTags.WOLF_FOOD, ItemTags.CAT_FOOD,
+                        ItemTags.HORSE_FOOD, ItemTags.CAMEL_FOOD, ItemTags.ARMADILLO_FOOD,
+                        ItemTags.BEE_FOOD, ItemTags.CHICKEN_FOOD, ItemTags.FROG_FOOD, ItemTags.HOGLIN_FOOD,
+                        ItemTags.LLAMA_FOOD, ItemTags.OCELOT_FOOD, ItemTags.PANDA_FOOD, ItemTags.PIG_FOOD,
+                        ItemTags.RABBIT_FOOD, ItemTags.STRIDER_FOOD, ItemTags.TURTLE_FOOD, ItemTags.PARROT_FOOD, ItemTags.AXOLOTL_FOOD);
```

Comment:
Could you please re-order this to be alphabetical? May make it much easier for people to check for what animal is included